css-layout-classes/index.html

```diff
@@ -8,49 +8,46 @@
 
   <body>
 
-    <div id="body">
-      <div id="title">
+    <div class="container">
+      <div class="col-full">
```

Comment:
`col-full` should be a child to `row`. So you need to move `col-full` into a `div` with a class of `row`